decidim-proposals/app/commands/decidim/proposals/admin/create_proposal_note.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # A command with all the business logic when an admin creates a private note proposal.
+      class CreateProposalNote < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # proposal     - A Decidim::Proposals::Proposal object.
+        # current_user - The current user.
```

Comment:
Param docs don't match params. Maybe it's easier to inject the proposal and current user to the form?